src/_scss/pages/trainingVideos/_listOfVideos.scss

```diff
@@ -31,6 +30,12 @@
       }
 
     }
+    .video-card__headline{
+      &:hover{
+        text-decoration: underline;
+        cursor: pointer;
+      }
+    }
     .list-of-videos__inline{
       display:inline-flex;
       width: 100%;
```

Comment:
at line 46 in this file you can remove the width: 100%
the width: 24 is being applied